assets/scss/_sidebar-tree.scss

```diff
@@ -0,0 +1,132 @@
+//
+// Left side navigation
+//
+.td-sidebar-nav {
+    padding-right: 0.5rem;
+    margin-right: -15px;
+    margin-left: -15px;
+
+    @include media-breakpoint-up(md) {
+        @supports (position: sticky) {
+            max-height: calc(100vh - 10rem);
+            overflow-y: auto;
+        }
+    }
+
+
+    @include media-breakpoint-up(md) {
+        display: block !important;
+    }
+
+
+    &__section {
+        li {
+            list-style: none;
+        }
+
+        ul {
```

Comment:
Will this also need to be updated to `li > ul` or an additional added with the same definition?

Reply:
Here's a preview with the changes from this PR: https://deploy-preview-235--zencartdocs.netlify.app/

Reply:
Effects of the change results in similar formatting and operation appears like it continues to function.  Doesn't fully resolve my device display issue of #202, though I note there is a `<style>` tag that is also providing invalid html; however, I was trying to go for the "meaty" fixes first. :)

I see no issues with the modifications that have been proposed through the link that was provided.